OsmAnd/src/net/osmand/plus/OsmAndFormatter.java

```diff
@@ -165,7 +165,17 @@ public static String getFormattedAlarmInfoDistance(OsmandApplication app, float
 
 		return df.format(meters / mainUnitInMeters) + " " + app.getString(mainUnitStr);
 	}
-	
+
+	public static String getFormattedAzimuth(float bearing) {
```

Comment:
Not sure about formatting, needs to be checked how it is usually formatted

Reply:
From Wiki: "Today, the reference plane for an azimuth is typically true north, measured as a 0° azimuth, though other angular units (grad, mil) can be used. Moving clockwise on a 360 degree circle, east has azimuth 90°, south 180°, and west 270°."